wireguard-go-rust-wrapper/wireguard-go/build.ps1

```diff
@@ -9,7 +9,7 @@ $ErrorActionPreference = "Stop"
 $env:CC="clang.exe";
 
 # WORKAROUND: Rust on Windows will look for LIBs without "lib" prefix, so we're naming ours "wireguard-go.lib"
-& go build -ldflags=-w -v -buildmode c-archive -o "$env:OUT_DIR\wireguard-go.lib" wireguard-go
+& go build -buildvcs=false -ldflags=-w -v -buildmode c-archive -o "$env:OUT_DIR\wireguard-go.lib" wireguard-go
```

Comment:
why is this flag added?

Reply:
`go build` can use git to grab a hash or some other description of the commit to put some metadata into the built lib. It fails in docker container on GH because, the user running in docker is different then the onc who cloned the repo and hance is the owner of the files. It could be fixed I guess be either `chown` or adding `safe.directory` to the git config, but I've decided not to do that since we don't need this built metadata in the lib anyway.